README.md

```diff
@@ -661,123 +658,99 @@ my-mbed-os-example (a5ac4bf2e468)
 
-1. Create a new empty repository on the remote site. This can be based on a public repository hosting service (GitHub, Bitbucket, mbed.org), your own service or even a different location on your system.
-2. Copy the URL/location of the new repository in your clipboard and open command-line in the local repository directory (for example change directory to `mbed-os-example/local-lib`).
-3. To associate the local repository:
- * For Git - run `git remote add origin <url-or-paht-to-your-remote-repo>`.
- * For Mercurial - edit .hg/hgrc and add (or replace if exists):
+1. Create a new empty repository on the remote site. This can be on a public repository hosting service (GitHub, Bitbucket, mbed.org), your own service or a different location on your system.
+1. Copy the URL/location of the new repository in your clipboard. 
+1. Open command-line in the local repository directory (for example, change directory to `mbed-os-example/local-lib`).
+1. To associate the local repository:
+ * For Git, run `git remote add origin <url-or-paht-to-your-remote-repo>`.
+ * For Mercurial, edit .hg/hgrc and add (or replace if exists):
  
             ```
             [paths]
             default = <url-or-paht-to-your-remote-repo>
             ```
-4. Run `mbed publish` to publish your changes.
+
+1. Run `mbed publish` to publish your changes.
 
 In a scenario with nested local repositories, start with the leaf repositories first.
 
 ### Forking workflow
 
-Git enables asymmetric workflow where the publish/push repository may be different than the original ("origin") one. This allows new revisions to land in a fork repository while maintaining an association with the original repository.
-
-To achieve this, first import an mbed OS program or mbed OS itself and then associate the push remote with your fork. For example:
+Git enables a workflow where the publish/push repository may be different than the original ("origin") one. This allows new revisions in a fork repository while maintaining an association with the original repository. To use this workflow, first import an mbed OS program or mbed OS itself, and then associate the push remote with your fork. For example:
 
 ```
 $ git remote set-url --push origin https://github.com/screamerbg/repo-fork
 ```
 
-Each time you `git` commit and push, or use `mbed publish`, the new revisions will be pushed against your fork. You can fetch from the original repository using `mbed update` or `git pull`. If you explicitly want to fetch or pull from your fork, then you can use `git pull https://github.com/screamerbg/repo-fork [branch]`.
+Each of `git commit`, `git push` and `mbed publish` push the new revisions to your fork. You can fetch from the original repository using `mbed update` or `git pull`. If you explicitly want to fetch or pull from your fork, then you can use `git pull https://github.com/screamerbg/repo-fork [branch]`.
```

Comment:
This isn't quite right. `git commit` and `git push` are the equivalent of `mbed publish`. So they can't be flattened to a list. The two `git` commands must remain paired.

So perhaps 

> Both `git commit & git push` and `mbed publish` push the new revisions...

The `&` is valid command syntax, I'm not using it as an English shortcut :)